analysis/analysis/metrics.py

```diff
@@ -1,24 +0,0 @@
-# -*- coding: utf-8 -*-
-
-"""
-Metric generation
-"""
-
-from os.path import basename, splitext
-
-import statsd
-
-
-def setup_metrics(host="localhost", name=None):
-    """Setup metric generation. Use dotted namespaces e.g.
-    "pipeline.centrifugation"
-    """
-    if name is None:
-        import __main__
-
-        prefix = splitext(basename(__main__.__file__))[0]
-    else:
-        prefix = name
-
-    prefix = prefix.strip(".")
-    return statsd.StatsClient(host, 8125, prefix=prefix)
```

Comment:
I gather that this file was unused, right?

Reply:
Oh, this actually needs to be re-added. It's actually being imported.

Reply:
Done